configurations/derived_products/item_10.yaml

```diff
@@ -0,0 +1,147 @@
+
+  - product: ls7_nbar_albers
+    name: ITEM
+    measurements: [blue, green, red, nir, swir1, swir2]
+    group_by: solar_day
+#    source_filter:
+#      product: ls7_level1_scene
+#      gqa_cep90: (-0.25, 0.25)
+    masks:
+      - product: ls7_pq_albers
+        measurement: pixelquality
+        group_by: solar_day
+        fuse_func: datacube.helpers.ga_pq_fuser
+        flags:
+          contiguous: True
+          cloud_acca: no_cloud
+          cloud_fmask: no_cloud
+          cloud_shadow_acca: no_cloud_shadow
+          cloud_shadow_fmask: no_cloud_shadow
+          blue_saturated: False
+          green_saturated: False
+          red_saturated: False
+          nir_saturated: False
+          swir1_saturated: False
+          swir2_saturated: False
+
+
+  - product: ls8_nbar_albers
+    name: ITEM
+    measurements: [blue, green, red, nir, swir1, swir2]
+    group_by: solar_day
+#    source_filter:
+#      product: ls8_level1_scene
+#      gqa_cep90: (-0.25, 0.25)
+    masks:
+      - product: ls8_pq_albers
+        measurement: pixelquality
+        group_by: solar_day
+        fuse_func: datacube.helpers.ga_pq_fuser
+        flags:
+          contiguous: True
+          cloud_acca: no_cloud
+          cloud_fmask: no_cloud
+          cloud_shadow_acca: no_cloud_shadow
+          cloud_shadow_fmask: no_cloud_shadow
+          blue_saturated: False
+          green_saturated: False
+          red_saturated: False
+          nir_saturated: False
+          swir1_saturated: False
+          swir2_saturated: False
+
+
+## Define whether and how to chunk over time
+date_ranges:
+  start_date: 2013-01-01
+  end_date: 2015-01-01
+  stats_duration: 2y
+  step_size: 2y
+  type: simple
+
+## Define output directory and file structure
+location: '/g/data/v10/ITEM/junk'
+
+input_region:
+  from_file: /g/data/v10/ITEM/ITEMv2_tidalmodel.shp
+#
+storage:
+  driver: NetCDFCF
+  #driver: Geotiff
+
+  crs: EPSG:3577
+  tile_size:
+          x: 100000.0
+          y: 100000.0
+  resolution:
+          x: 25
+          y: -25
+  chunking:
+      x: 200
+      y: 200
+      time: 1
+  dimension_order: [time, y, x]
+
+
+## Define statistics to perform and how to store the data
+
+output_products:
+ - name: med_ndwi
+   statistic: medndwi
+   output_params:
+      zlib: True
+      fletcher32: True
+   file_path_template: '{x}_{y}_{epoch_start:%Y%m%d}_{epoch_end:%Y%m%d}_{stat_name}.nc'
+   product_type: ITEM
+
+# - name: standard_deviation 
+#   statistic: std
+#   output_params:
+#      zlib: True
+#      fletcher32: True
+#   file_path_template: '{x}_{y}_{epoch_start:%Y%m%d}_{epoch_end:%Y%m%d}_{stat_name}.nc'
+#   product_type: ITEM
+
+# - name: tidal_count
+#   statistic: clearcount
+#   output_params:
+#      zlib: True
+#      fletcher32: True
+#   file_path_template: 'PIXEL_COUNT_{x}_{y}_{epoch_start:%Y%m%d}_{epoch_end:%Y%m%d}.nc'
+#   product_type: ITEM
+#
+filter_product:
+  method: by_tide_height
+  args:
+     name: item
```

Comment:
How is this `name` used? I'm not convinced it's necessary, since this whole configuration file is about **ITEM**. Unless it's triggering something other than a name somewhere?